mitiq/lre/tests/test_lre.py

```diff
@@ -101,20 +101,6 @@ def test_lre_executor_with_chunking():
     assert abs(lre_exp_val - ideal_val) <= abs(noisy_val - ideal_val)
 
 
-@pytest.mark.parametrize("num_chunks", [(1), (2), (3), (4), (5), (6), (7)])
```

Comment:
the test that keeps failing is when `num_chunks=7`. 

I think it might be better to remove that particular one instead of the entire test.

Reply:
If we want to keep the test, I think we should re-work the test so that it is not testing performance, but correctness. What would you suggest changing the assertion to?

Reply:
Purva and I spoke about this live and decided that it was okay to remove. Purva was partially using the unit test to track which values were causing issue, but this is now at least partially recorded here.

Going ahead with removal.